V1/Services/TestResultService.cs

```diff
@@ -73,6 +76,9 @@ public async Task AddTestResult(TestResultRequest testResultRequest)
             await _walletTestResultRepository.AddAsync(testResults);
 
             await _walletTestResultRepository.SaveAsync();
+
+            if (testResultRequest.ResultStatus == ResultStatus.Positive)
+                BackgroundJob.Enqueue(() => _staySafeService.CaptureData(wallet.Id, testResultRequest.TestedAt));
```

Comment:
Just add this to the other AddTestResult function. Or maybe just get rid of the second and adjust the call from the OTP side

Reply:
I don't think we are using that flow on the frontend to submit the wallet and test results in the otp at once. But ill update it